packages/core/build/tokens.ts

```diff
@@ -344,6 +345,16 @@ const typography = {
   topGapHeight: token('0.1475em'), // line-height eraser
   ascenderHeight: token('0.1703em'), // line-height eraser
   xHeight: token('0.517em'), // line-height eraser
+  link: {
+    color: {
+      value: token(color.blue[800]),
+      hover: token(color.blue[900]),
+      visited: {
+        value: token(color.lavender[600]),
+        hover: token(color.lavender[700]),
+      },
+    },
```

Comment:
I believe the `cds-inline-button` colors have not been updated accordingly:

<img width="804" alt="Screen Shot 2021-02-05 at 3 08 28 AM" src="https://user-images.githubusercontent.com/113730/106971204-6c616700-671c-11eb-8e8c-d8d65fa060d4.png">